packages/react-native/React/Base/RCTConvert.m

```diff
@@ -1538,13 +1637,13 @@ + (NSString*)accessibilityRoleFromTrait:(NSString*)trait
   return role;
 }
 
-+ (NSString *)accessibilityRoleFromTraits:(id)json
++ (NSString *)accessibilityRoleFromTraits:(id)json withAriaMappings:(BOOL) useAriaMappings
```

Comment:
nit: remove extra space before `useAriaMappings`, as well as before `json` in the two invocations to `accessibilityRoleFromRole:` below

Reply:
Thanks, I'm still getting used to the style conventions in Objective C